etc/kworkflow_template.config

```diff
@@ -23,6 +23,10 @@ ssh_port=22
 # Specify the default architecture used by KW
 arch=x86_64
 
+# Specify the bootloader being used.
+# Currently supported targets are `grub` (grub2) and `rpi_bootloader` (Raspberrypi native bootloader)
+bootloader=grub
```

Comment:
I'm not sure if I like that... I think kw should be smart enough to detect the bootloader instead of waiting for the user input. Also, enable users to specify the bootloader opens a door for mistakes.